### ISSUE TITLE
Store API key in statically located file

### ISSUE THREAD

**Issue by NikolaiMagnussen** [NikolaiMagnussen/Git-Gud-tool#5]
I think it would be a good idea to add the capability of not having the API key in either config.ini or environment, but a different file. 

I propose adding a subcommand which will allow a user to set an API key that will be stored in a (hidden?) file (probably in the home directory) that is "protected" by the file system (chmod 400).

This should not supercede the environment variable, but will simply store the API key in a file that will be user specific, making it useful regardless of where you are running the file from, resulting in the same flexibility and ease of use.

**Comment by emersonmello** [comment]
Hi @NikolaiMagnussen,

I agree that we should not store credentials in plaintext. However, chmod 400 will not help either.

On macOS we should use osxkeychain (I already use it to store my github credentials). There is Gnome Keyring for Linux and Windows Credential Locker for Windows.

I found it: https://pypi.org/project/keyring/ and maybe it could be a good option.

First of all you have to store GITHUB TOKEN on keychain:
```python
import keyring
keyring.set_password("system", "gitgudtoken", "YOUR API KEY HERE")
```

Thus, you have to modify your current `git_gud.py`:

```python
#!/usr/bin/env python3
  
import os
import sys
import subprocess
from github import Github, GithubException
import configparser
import keyring

# Get TOKEN from osxkeychain
_key = keyring.get_password("system", "gitgudtoken")

if _key is None:
    # Try to get it from system environment variable (On Linux/macOS you have to create it on $HOME/.bash_profile or .bashrc)
    _key = os.environ.get('GIT_GUD_TOKEN', None)
...
...
...
...
```

Cheers

**Comment by NikolaiMagnussen** [comment]
That is true, and using the keyring system to store credentials is a great idea.

I did not know of that Python library, and that it could be so easy.

I'll take a look at it and potentially implement it after Dec. 15.
